src-docs/src/views/resizable_container/resizable_container_reset_values.js

```diff
@@ -18,8 +20,8 @@ const text = (
   </>
 );
 
-const firstPanelId = 'resizable-panel__1';
-const secondPanelId = 'resizable-panel__2';
+const firstPanelId = htmlIdGenerator('firstPanel')('first');
+const secondPanelId = htmlIdGenerator('secondPanel')('second');
```

Comment:
Probably don't need `first` twice?
```suggestion
const firstPanelId = htmlIdGenerator('firstPanel')();
const secondPanelId = htmlIdGenerator('secondPanel')();
```